### ISSUE TITLE
Pancolitis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6188]
**Preferred term label:**
Pancolitis

**Synonyms**
Pancolitis

**Definition (free text, please give PubMed ID)**
 Inflammation of the entire colon.
DOI: https://doi.org/10.1097/01.mib.0000179469.86500.ac

**Parent term (use hpo.jax.org/app)**
Ulcerative colitis

**Comment by pnrobinson** [comment]
This term is needed for this disease
https://omim.org/entry/613148
PMID:2151936 1/1, age of onset infantile

**Comment by johannaest** [comment]
​Yes, Pancolitis can be a phenotype of inflammatory bowel disease.



Von: Peter Robinson <notifications@github.com>
Gesendet: Freitag, 9. Oktober 2020 12:55
An: obophenotype/human-phenotype-ontology
Cc: Est, Johanna Ligaya; Author
Betreff: Re: [obophenotype/human-phenotype-ontology] Pancolitis (#6188)
 



This term is needed for this disease
https://omim.org/entry/613148
PMID:2151936 1/1, age of onset infantile
—
You are receiving this because you authored the thread.
Reply to this email directly, 
view it on GitHub, or 
unsubscribe.

**Comment by pnrobinson** [comment]
added
